### ISSUE TITLE
Reading suggestions or autocomplete of extensions

### ISSUE THREAD

**Issue by jeanbrazcosta** [microsoft/vscode#183030]
Type: <b>Feature Request</b>

Depending on the extension we are using, when code suggestion information appears, it is not read by screen readers.

Steps to reproduce.

Install the codeium extension.

Open any project, and start typing or commenting the code;

You will hear a sound when a hint appears, but this hint is not read by screen readers like nvda.

I suggest that it is possible to activate a configuration in vscode, so that this type of information can be transmitted to screen readers.

VS Code version: Code - Insiders 1.79.0-insider (f6be5461f8bc69013a605f5baea834651c6589fb, 2023-05-20T01:18:53.811Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

**Comment by meganrogge** [comment]
I can reproduce. We must not be getting the line content here https://github.com/microsoft/vscode/blob/43cf22b41a7d999b3bc7b535bf8db093fa05e615/src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts#L188-L189

**Comment by jeanbrazcosta** [comment]
@meganrogge I would like to inform you that I also tested it with the copilot and when listening to the sound that a suggestion was given, the screen readers also do not read.

**Comment by meganrogge** [comment]
You need to set `editor.screenReaderAnnounceInlineSuggestion` to `true` and it will work for copilot @jeanbrazcosta

**Comment by meganrogge** [comment]
@jooyoungseo do you think we should now set that to `true` by default?

**Comment by jeanbrazcosta** [comment]
Legal @meganrogge I defined it and it actually worked for the copilot.

My suggestion is that it would not be possible for these settings to go to the accessibility feature?

I know that it makes sense to stay where they are because it has to do with the editor, but they are difficult to find, or maybe we have some dock where all the accessibility features are and their respective paths where they should be activated, you get the idea.

**Comment by meganrogge** [comment]
Yes, we actually have help menus now for the copilot chat view / interactive editor session. When you focus those inputs, using `alt+f1` reveals/focuses the help menu for each. 

I think for this case, we should enable inline suggestion by default

**Comment by jeanbrazcosta** [comment]
Sorry, @meganrogge just found out that by enabling the setting, it also worked for codeium.

Should I then close this feature?

**Comment by meganrogge** [comment]
I think we should enable this by default, so will use this issue to track that

**Comment by jooyoungseo** [comment]
@meganrogge I agree with the idea setting the inline suggestion announcement to TRUE by default as it is now stable.

**Comment by Tyriar** [comment]
Verified the setting default is flipped
